resources/config.edn

```diff
@@ -22,13 +22,17 @@
                                     "taskid" #or [#env MESOS_TASK_ID #rand-uuid "rand"]}
                              :seconds #profile {:staging 60
                                                 :local 1}}}
- :logging {:level :info ; e/o #{:trace :debug :info :warn :error :fatal :report}
+ :logging {:level :error ; e/o #{:trace :debug :info :warn :error :fatal :report}
```

Comment:
Anything lower than this has been completely ruined by kafka et el now. Trying to find a selective appender setting in timbre...